src/parameter_entry_class.F90

```diff
@@ -137,6 +137,11 @@
 !!      inner lists are the columns of the matrix. No newlines are are written.
 !!
 
+#ifdef GNU_54070
+! GNU_54070 is the same bug as INTEL_DPD200255963, so we just reuse the define.
+#  define INTEL_DPD200255963
```

Comment:
I'd prefer not to equate the two.  Better below to say #ifdef INTEL_DPD200255963 || GNU_54070  (or whatever the proper syntax is).